test/base/zero-shot-image-classification.spec.ts

```diff
@@ -47,4 +40,17 @@ describe("ZeroShotImageClassification", () => {
       expect.any(Object),
     );
   });
+
+  it("should send a request with base64 image", async () => {
+    const response = await model.generate({
+      image: fakeFileBuffer as Buffer,
+      candidate_labels: ["dog", "cat"],
+    });
+    expect(response).toBeDefined();
+    expect(postMock).toHaveBeenCalledWith(
```

Comment:
I guess you can do a better job at asserting here something more meaningful. Also the test name is bogus.